firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -66,13 +73,19 @@
   private ShadowPackageManager shadowPackageManager;
 
   @Mock private FirebaseInstallationsApi mockFirebaseInstallations;
+  @Mock private FirebaseAppDistributionTesterApiClient mockFirebaseAppDistributionTesterApiClient;
+  @Mock private InstallationTokenResult mockInstallationTokenResult;
```

Comment:
AppDistributionRelease is a simple object. You don't need to use Mockito. You should build the AppDistributionRelease using the AppDistributionReleaseBuilder.